api/ccapi/include/tensor_dim.h

```diff
@@ -65,15 +121,41 @@ class TensorDim {
    * formats of {w}, {h, w}, {c, h, w}, {b, c, h, w} for the NCHW & NHWC are
    * accepted
    */
-  TensorDim(std::initializer_list<size_t> dims, Format fm = Format::NCHW);
+  TensorDim(std::initializer_list<size_t> dims,
+            TensorType t_type_ = TensorType());
+
+  // TensorDim(std::initializer_list<size_t> dims, TensorDim::Format
```

Comment:
Should we remove the commented code OR add a revisit comment?

Reply:
I will remove. thanks.